examples/use-cases/Kernel2Wireguard2Kernel/README.md

```diff
@@ -66,7 +66,7 @@ spec:
         - name: nsc
           env:
             - name: NSM_NETWORK_SERVICES
-              value: kernel://icmp-responder/nsm-1
+              value: kernel://icmp-responder-wireguard/nsm-1
```

Comment:
I think `icmp-responder-ip` will be better. Because `nsc` doesn't know about wireguard

Reply:
ok, done